### PR TITLE
plugins.xunit: Python3 fixes

### PR DESCRIPTION
**Pull request by ldoktor**
For a long time I don't see outputs of some tests and finally got to look at it. Spoiler alert, it's py3k fix :smile:

### CODE REVIEW

File: avocado/plugins/xunit.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def _create_failure_or_error(self, document, test, element_type,
                         text_output = logfile_obj.read(size)
                         text_output += ("\n\n--[ CUT DUE TO XML PER TEST "
                                         "LIMIT ]--\n\n")
-                        logfile_obj.seek(-size / 2, 2)
+                        logfile_obj.seek(-size, 2)
```
Comment (clebergnu) [inline avocado/plugins/xunit.py]: OK.

File: avocado/plugins/xunit.py [hunk 2]
```diff
@@ -71,7 +71,7 @@ def _create_failure_or_error(self, document, test, element_type,
                     if log_size < max_log_size:
                         text_output = logfile_obj.read()
                     else:
-                        size = max_log_size / 2
+                        size = int(max_log_size / 2)
```
Comment (clebergnu) [inline avocado/plugins/xunit.py]: This works fine, but one other way that avoids the type conversion is to use `//`.

File: avocado/plugins/xunit.py [hunk 3]
```diff
@@ -76,7 +76,7 @@ def _create_failure_or_error(self, document, test, element_type,
                         text_output = logfile_obj.read(size)
                         text_output += ("\n\n--[ CUT DUE TO XML PER TEST "
                                         "LIMIT ]--\n\n")
-                        logfile_obj.seek(-size, 2)
+                        logfile_obj.seek(log_size - size, 0)
```
Comment (clebergnu) [inline avocado/plugins/xunit.py]: You're right, that ends up in:

```
io.UnsupportedOperation: can't do nonzero end-relative seeks
```

File: avocado/plugins/xunit.py (hunk 2 above)
Comment (ldoktor) [reply avocado/plugins/xunit.py]: oh, true... That would have been better...

**Pull request merged**